app/controllers/application_controller/buttons.rb

```diff
@@ -152,14 +151,8 @@ def ab_button_delete
     if custom_button.parent
       automation_set = CustomButtonSet.find_by(:id => custom_button.parent.id)
       if automation_set
-        mems = automation_set.members
-        if mems.length > 1
-          mems.each do |m|
-            automation_set.remove_member(custom_button) if m.id == custom_button
```

Comment:
funnily enough, comparing integers and CustomButton instances probably didn't work